src/components/Table.js

```diff
@@ -111,6 +111,7 @@ const Table = ({ data }) => {
   const filteredItems = filterText
     ? rows.filter(
         row =>
+          row.name && row.name.toLowerCase().includes(filterText.toLowerCase()) ||
```

Comment:
Regarding the keyword search, what other fields would a user search other than place and name? I figure the expectation of searching for any field is essentially correct in this use case. Placeholder "Filter by place or name" was simply too long to display on viewports above 700px and frankly felt a bit overkill.

I would appreciate it if you could elaborate as to the issues this method would have with scale as I'm not currently aware. To be fair, I only spent as long on this as it took to find the person I was searching for but through limited testing, I saw that both row.work and row.name were searched and filtered on the page despite the priority. I may be ignorant of the consequence of this specific implementation but the results were as intended as seen below

![Screen Shot 2020-04-06 at 2 38 32 PM](https://user-images.githubusercontent.com/45721766/78608285-78874a00-7815-11ea-81a6-04908a5faf3f.png)

Reply:
Happy to elaborate. Potentially, any column is one that might be worth filtering on: e.g. show me all the people who accept venmo. This brings me to my comment about scale, i.e. if more columns are added that you wanted users to be able to search, you would need to add more conditions to that filter callback, e.g.
```
row.name && row.name.toLowerCase().includes(filterText.toLowerCase()) ||
row.work && row.work.toLowerCase().includes(filterText.toLowerCase()) ||
row.somePropA && row.somePropA.toLowerCase().includes(filterText.toLowerCase()) ||
row.somePropB && row.somePropB.toLowerCase().includes(filterText.toLowerCase()) ||
row.somePropC && row.somePropC.toLowerCase().includes(filterText.toLowerCase()) ||
row.somePropD && row.somePropD.toLowerCase().includes(filterText.toLowerCase()) ||
```
and so on, which is not particularly readable and contains a lot of duplication, which is generally to be avoided.

Tell me which you think is more readable and scales regardless of the number of properties in the row object . The above or this:
```
const filteredRows = rows.filter((row) => Object.values(row)
  .map(r => r.toLowerCase())
  .includes(filterText));
```
Also, I'm inclined to agree about priority since implementing real matching and prioritization is beyond the scope of this simple filter.

Reply:
Thank you for the explanation! I had interpreted your concern to be about scale issues with growth of the dataset rather than literal lines of code for each case - I fully agree with you there.

I think the best of both worlds would be to allow users the ability to sort ascending/descending 
 with a toggle click on the Pay, support, and healthcare fields while using the input search for names and places. This doesn't solve for edge cases like "I want to find people with venmo and without healthcare" but is a step in the right direction in my opinion. 

Nevertheless, my intention was just to add functionality for filtering by names because it isn't currently possible. I can change the placeholder text to "Filter by Name or Place" which actually fits perfectly or close the PR altogether. 

I meant to help with something I thought was overlooked rather than create more work for you so my apologies for any inconvenience!

Reply:
Thanks for the contribution @IsaacCDHall and for the review @marciaga ! Yeah, I like going with `Filter by name or place` for the sake of clarity here. Adding it as a suggestion below.